index.html

```diff
@@ -20,56 +20,287 @@
+            <a href="#" class="btn-menu-toggle"><i class="fas fa-bars"></i></a>
+          </nav>
         </div>
-        <nav class="nav-menu">
-          <a href="#" class="btn-menu-toggle"><i class="fas fa-bars"></i></a>
-        </nav>
-      </div>
-    </header>
-    <section class="headline">
-      <div class="container">
-        <h1 class="headline--title">I’m Tekle</h1>
-        <h2 class="headline--title subtitle">Glad to see you!</h2>
-        <p class="headline--body">
-          I’m a software developer! I can help you build a product , feature or
-          website Look through some of my work and experience! If you like what
-          you see and have a project you need coded, don’t hestiate to contact
-          me.
-        </p>
+      </header>
+      <section class="headline">
+        <div class="container flex-container">
+          <h1 class="headline--title">I’m Tekle</h1>
+          <h2 class="headline--title subtitle">Glad to see you!</h2>
+          <p class="headline--body">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+          <p class="headline--supporting-text">Let's connect</p>
+          <ul class="social-media">
+            <li class="social-meida__list-item">
+              <a
+                href="https://twitter.com/teklegyohannes"
+                class="social-media-items twitter-link"
+              >
+                <i class="fab fa-twitter"></i>
+              </a>
+            </li>
+            <li class="social-meida__list-item">
+              <a
+                href="https://www.linkedin.com/in/tekle-gebreyohannes-kidanemariam-7605752b"
+                class="social-media-items linkedin-link"
+              >
+                <i class="fab fa-linkedin-in"></i>
+              </a>
+            </li>
+            <li class="social-meida__list-item">
+              <a href="#" class="social-media-items medium-link">
+                <i class="fab fa-medium-m"></i>
+              </a>
+            </li>
+            <li class="social-meida__list-item">
+              <a
+                href="https://github.com/gtekle"
+                class="social-media-items github-link"
+              >
+                <i class="fab fa-github"></i>
+              </a>
+            </li>
+            <li class="social-meida__list-item">
+              <a href="#" class="social-media-items angelist-link">
+                <i class="fab fa-angellist"></i>
+              </a>
+            </li>
+          </ul>
+        </div>
+      </section>
+    </div>
 
-        <p class="headline--supporting-text">Let's connect</p>
+    <section class="works">
+      <div class="container grid-container">
+        <h2 class="section-title">Projects</h2>
+        <article class="card">
```

Comment:
- [ ] (Optional) Putting this makes the project really nice. It would be nicer to make the text bigger. I think you can make the `font-size: 40px`